.travis.yml

```diff
@@ -11,8 +11,3 @@ install:
 script:
   - npm run test:server
   - npm run test:ui
-
-deploy:
-  provider: azure_web_apps
-  on: develop
-  verbose: true
```

Comment:
is there a particular reason we removed this?

Reply:
It says deployment is not done in travis run every-time. We are good in running test cases but deployment is not configured properly yet with our ever changing scenarios. So removed it, as without clearing this step I cannot merge where as that is not configured properly.